src/main/scala/cromwell/util/IntUtil.scala

```diff
@@ -0,0 +1,18 @@
+package cromwell.util
+
+object IntUtil {
+
+  val OrdinalSuffixes = List("th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th")
+
+  implicit class IntEnhanced(val value: Int) extends AnyVal {
+    def toOrdinal: String = value match {
+      case v if v.isBetweenInclusive(10, 20) => s"${v}th"
```

Comment:
Will it be removed from Cromwell in that case? It'd be nice to do that before merging a duplicate